### DISCUSSION TITLE
debugging BSP connector - log stderr?

### DISCUSSION THREAD

**Discussion by tdyas** [scalameta/metals#3657] [General]
Is there any way to log the stderr output from a BSP connector to `metals.log` or a trace file (similar to how `bsp.trace.json` is logged)?

**Comment by tdyas** [comment]
For now, I'm just using a stub shell script to redirect stderr, but thought I would ask in case I'm missing a more convenient logging tool.

**Comment by tgodzik** [comment]
Thanks for the question! stderr from BSP connector should actually go into metals.log. If it doesn't it's most likely a bug, so if you have a reproduction we should be able to take a look into it.
